capstone/src/main/java/com/google/step/search/RestaurantHeaderSearchClient.java

```diff
@@ -0,0 +1,38 @@
+package com.google.step.search;
+
+import com.google.step.data.RestaurantHeader;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * The {@code RestaurantHeaderSearchClient} sends requests to add, update and query
+ *  {@code RestaurantHeaders} in a search Index.
+ */
+public interface RestaurantHeaderSearchClient {
+
+  /**
+   * Given a {@link RestaurantHeader}, updates search index with a document representing the
+   * restaurant header. If the document for the restaurant header does not currently exist a new
+   * document will be created. Otherwise, a new document is created.
```

Comment:
"If the document for the restaurant header does not currently exist a new document will be created. Otherwise, a new document is created."

Consider rewording this comment